zebra/zserv.c

```diff
@@ -2461,13 +2461,16 @@ static inline void zread_rule(ZAPI_HANDLER_ARGS)
 		STREAM_GET(&zpr.filter.dst_ip.u.prefix, s,
 			   prefix_blen(&zpr.filter.dst_ip));
 		STREAM_GETW(s, zpr.filter.dst_port);
+		STREAM_GETL(s, zpr.filter.fwmark);
```

Comment:
Where is the stream_putl for this ?

Reply:
it is in BGPd.
I guess the same is done for PBRd.

where is the stream_putl for zpr.filter.dst_port ?